cms/templates/admin/cms/page/history/revision_header.html

```diff
@@ -1,22 +1,22 @@
 {% extends "admin/cms/page/change_form.html" %}
-{% load i18n %}
+{% load i18n cms_tags admin_urls %}
 
 {% block breadcrumbs %}
-<div class="breadcrumbs">
-    <a href="../../../../../">{% trans "Home" %}</a> &rsaquo;
-    <a href="../../../../">{{ app_label|capfirst|escape }}</a> &rsaquo;
-    <a href="../../../">{{ opts.verbose_name_plural|capfirst }}</a> &rsaquo;
-    <a href="../../">{{ original|truncatewords:"18" }}</a> &rsaquo;
-    <a href="../">{% trans "History" %}</a> &rsaquo;
-    {% blocktrans with opts.verbose_name as verbose_name %}Revert {{ verbose_name }}{% endblocktrans %}
-</div>
+    <div class="breadcrumbs">
+        <a href="{% url 'admin:index' %}">{% trans "Home" %}</a> &rsaquo;
+        <a href="{% url 'admin:app_list' app_label=opts.app_label %}">{{ opts.app_config.verbose_name }}</a> &rsaquo;
+        <a href="{% url opts|admin_urlname:'changelist' %}">{{ opts.verbose_name_plural|capfirst }}</a> &rsaquo;
+        <a href="{% url opts|admin_urlname:'recoverlist' %}">{{ original|truncatewords:"18" }}</a> &rsaquo;
+        <a href="../">{% trans "History" %}</a> &rsaquo;
```

Comment:
I got it to work for recoverlist @yakky just recover remains. I found the code ``reversion``. Dunno though where this view is used.